backend/src/API/mailAPI.ts

```diff
@@ -194,3 +195,26 @@ export const sendTECReminder = async (
   } pending this semester.\n${reminder}\nTo submit your TEC, please visit https://idol.cornelldti.org/forms/teamEventCredits.`;
   return emailMember(req, member, subject, text);
 };
+
+/**
+ * Send an email reminder to members who do not have a coffee chat blackout
+ * @param req - The request made when sending the email
+ * @param member - The member being sent the email
+ * @returns - The response body containing information of the member being sent the email
+ */
+export const sendCoffeeChatReminder = async (
+  req: Request,
+  member: IdolMember
+): Promise<AxiosResponse> => {
+  const subject = 'Coffee Chat Reminder';
+
+  const isNewbie = await checkMemberMeetsCategory(member.email, member.email, 'a newbie').then(
+    (result) => result.status === 'pass'
+  );
+
+  const text = `Hey! You currently have no coffee chat bingo this semester
+  ${isNewbie ? ', and newbies are required to get at least 1 bingo' : ''}
+  .\nThis is a reminder to submit your coffee chats by the end of the semester.\n
+  To submit your coffee chats, please visit https://idol.cornelldti.org/forms/coffeeChats.`;
```

Comment:
nit: There's also a little indent in the message here, maybe we can remove and try to align the text on the left?